### PR TITLE
Add section How to read footprint of a cell to docs

### PR DESCRIPTION
**Pull request by Zachary-Ip**
Screenshot 2024-01-17 at 4 24 10 PM

- Added Doc
- updated mkdocs.yaml

### CODE REVIEW

**Comment by shayqn** [comment]
@Zachary-Ip it should read "... image data for the first cell" (not frame)

**Comment by shayqn** [comment]
@Zachary-Ip same correction should be made in the Indexing box

**Comment by shayqn** [comment]
@Zachary-Ip looks good to me.

File: docs/how-to/read-cellsets-footprint.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+```python
+footprint = cell_set.get_cell_image_data(0)
+```
+
+`footprint` now contains a 2D [numpy array](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html) with the image data for the first cell. 
+
+
+
+!!! warning "Indexing"
+    Note that python indexes by 0, so the first cell is at index 0, and the the second cell is at index 1, and so on.
```
Comment (nosheen-adil) [inline docs/how-to/read-cellsets-footprint.md:28]: maybe add that the last cell index is `n-1` where `n` is the number of cells in the cell set

**Review by nosheen-adil — APPROVED** [review]

**Pull request merged**